lago/utils.py

```diff
@@ -533,12 +534,12 @@ def load_virt_stream(virt_fd):
         dict: Loaded virt config
     """
     try:
-        virt_conf = json.load(virt_fd)
+        virt_conf = json.load(virt_fd, object_pairs_hook=collections.OrderedDict)
     except ValueError:
         virt_fd.seek(0)
         virt_conf = yaml.load(virt_fd)
 
-    return deepcopy(virt_conf)
+    return copy.deepcopy(virt_conf)
```

Comment:
I added to the deepcopy the ability to copy OrderedDict - but it still did not work for :(
Only when I moved to this implementation, it succeeded.
Any idea why?